doc_admin/AdminManual.adoc

```diff
@@ -389,11 +389,11 @@ include::https://raw.githubusercontent.com/informatici/openhospital/master/oh-bu
 
 Backup of the Open Hospital database can be performed in several ways.
 
-NOTE: Starting with version 1.11, the easiest way is to use the provided scripts (oh.sh/oh.bat) by choosing the option "-s".
+NOTE: Starting with version 1.11, the easiest way is to use the provided scripts: on Linux execute *oh.sh -s*, on Windows launch *oh.bat* and choose the option *s*.
 
 Restore of the Open Hospital database can be performed also in several ways.
 
-NOTE: Starting with version 1.11, the easiest way is to use the provided scripts (oh.sh/oh.bat) by choosing the option "-r".
+NOTE: Starting with version 1.11, the easiest way is to use the provided scripts: on Linux execute *oh.sh -r*, on Windows launch *oh.bat* and choose the option *r*.
```

Comment:
Didn't we agree not to use "Starting with version x.x"?   If we didn't then there are other places this needs to be added; for example, the DBMS configuration files were previously in *etc* and now they are in *data/db*.

Reply:
I already corrected the reference to the data/conf directory in place of the etc folder.
Agree on the 1.11.1 ref version, removed. We will still have to do another update before the 1.11.1 release to replace the 1.11.0 references still present.

Reply:
My point was more if we are highlighting changes in options to scripts happening why aren't we highlighting when changes take place to the moving of locations of files?   I see both types of changes happening at a particular release boundary but only one of them is specifically noted.     Just my 2 cents.

Reply:
I think to remove all refs in any case is the best thing and also more easy to manage.